classes/class-wc-connect-logger.php

```diff
@@ -38,25 +43,42 @@ protected function format_message( $message, $context = '' ) {
 
 		}
 
+		public function enable_logging() {
+			update_option( WOOCOMMERCE_CONNECT_DEBUG_LOGGING_ENABLED_OPTION, true );
+			$this->is_logging_enabled = true;
+			$this->log( "Logging enabled" );
+		}
+
+		public function disable_logging() {
+			$this->log( "Logging disabled" );
```

Comment:
very very minor but i had to double take here because the order of operations here is different then for enable